examples/complex_numbers.js

```diff
@@ -44,6 +44,36 @@ print(c);                      // 1 + i
 var d = math.complex(3, 4);
 console.log(d.abs(), d.arg());      // radius = 5, phi = 0.9272952180016122
 
+// comparisions between two complex numbers (only considers real components)
+console.log('\ncomparisions between two complex numbers (only considers real components)');
```

Comment:
https://github.com/josdejong/mathjs/pull/852/commits/4e324ee502148cdbf0768eed320601b37563dc6a should be concise enough